packages/EasyStandard/src/Rector/RestoreDefaultNullToNullableTypeParameterRector.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyStandard\Rector;
+
+use PhpParser\Node;
+use PhpParser\Node\NullableType;
+use PhpParser\Node\Param;
+use PhpParser\Node\Stmt\ClassMethod;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+use Rector\Core\ValueObject\PhpVersionFeature;
+
+final class RestoreDefaultNullToNullableTypeParameterRector extends AbstractRector
+{
+    public function getDefinition(): RectorDefinition
+    {
+        return new RectorDefinition('Add null default to parameters with PHP 7.4 nullable type', [
```

Comment:
Let's replace "parameters" with "function arguments".
And why do you mention PHP 7.4 here? It is not specific to the 7.4 version.

Reply:
Done. I replaced with PHP 7.1 because this feature is available since that version.